Content.Server/Sound/EmitSoundSystem.cs

```diff
@@ -65,15 +58,10 @@ private string SelectRandomSoundFromSoundCollection(string soundCollectionName)
 
         private static void PlaySingleSound(string soundName, BaseEmitSoundComponent component)
         {
-            if (string.IsNullOrWhiteSpace(soundName))
-            {
-                return;
-            }
-
-            if (component._pitchVariation > 0.0)
+            if (component.PitchVariation > 0.0)
```

Comment:
does this need an if/else? just pass the PitchVariation in, even if its 0, and it just shouldn't get varied if its 0

Reply:
correct, it will resolve to a scale of 1, not modyfying the pitch